tests/utilities/test_model_summary.py

```diff
@@ -139,19 +138,6 @@ def forward(self, inp):
         return self.head(self.branch1(inp), self.branch2(inp))
 
 
-def test_invalid_weights_summary():
-    """Test that invalid value for weights_summary raises an error."""
-    model = LightningModule()
-
-    with pytest.raises(
-        MisconfigurationException, match="`weights_summary` can be None, .* got temp"
-    ), pytest.deprecated_call(match="weights_summary=temp)` is deprecated"):
-        Trainer(weights_summary="temp")
```

Comment:
yes, this can be deleted, but not the part below

Reply:
Made the change, can you have a look